scripts/imu_spawners/spawn_estimated_imu.py

```diff
@@ -238,14 +238,33 @@ def load_estimated_poses(filename):
     poses = np.zeros((7, 7))
```

Comment:
Same here. Can we automatically read from `Controller`?

Reply:
yes

Reply:
done